docs/index.md

```diff
@@ -179,6 +180,78 @@ packageCatalog.PackageUpdating += OnPackageUpdating;
   Your browser does not support the video tag.
 </video>
 
+## Exercise 1.5: Embedded .AppInstaller
+
+### What does this feature do?
+
+The embedded .AppInstaller feature allows developers to deploy a single MSIX package while simultaneously retaining the features enabled by the .AppInstaller file format. This means that developers can easily configure automatic update settings for the package. This feature also provides the functionality of being able to install the .MSIX package even if the .AppInstaller URI is inaccessible. 
+
+### What is the magic sauce here?
+
+**For Windows 10, version 2104 and newer (Min Version):**
+
+From Windows 10 SDK 2104 onwards, we can make use of the UAP13 schema which provides the AutoUpdate feature highlighted in the Package.appxmanifest in the code below:
+
+```xml
+	<uap13:AutoUpdate>
+		<uap13:AppInstaller File="Update.appinstaller" />
+	</uap13:AutoUpdate>
+```
+
+Within the MyEmployees.Package itself, we embed the “Update.appinstaller” file with the settings we would like to update with and we can publish the package as normal. Essentially, users can install the package with the MSIX package and this property will associate the “Update.appinstaller” file when required.
+
+**For Windows 10, version 1809 and newer:**
+
+Since the UAP13 schema is not available, the AppInstaller file APIs are used to achieve similar results in the AddAppInstaller() function in Form1.cs. First, the GetAppInstallerInfo() method is used as a check to see if there is an AppInstaller file for the current package as shown below: 
```

Comment:
You might consider describing that the embedded .appinstaller file will not automatically update the MSIX package after install.